llama_stack/apis/agents/agents.py

```diff
@@ -426,6 +428,7 @@ async def create_agent(
     ) -> AgentCreateResponse: ...
 
     @webmethod(route="/agents/turn/create")
+    @traced(input="messages")
```

Comment:
remove this because it feels like a mixture of concerns at the API level

Reply:
done.